Dockerfile

```diff
@@ -4,4 +4,14 @@ FROM python:3.8.2-alpine
 COPY action_builder.sh /action_builder.sh
 COPY actions/ /actions
 
+RUN $FILES_ADDED > /files_added
+RUN $FILES_MODIFIED > /files_modified
+RUN $PR_BODY > /pr_body
+RUN $BRANCH_NAME > /branch_name
+
+RUN cat /files_added
+RUN cat /files_modified
+RUN cat /pr_body
+RUN cat /branch_name
```

Comment:
to make this as generic as possible, is better to push this into some sort of script to later on port it. With this particular piece of code, we are making this very one-off for this in particular